From 1eb7c40bcf7aeb359e9e605ebf2ae7e4a602a23c Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Fri, 24 Feb 2017 12:12:03 -0800
Subject: [PATCH 77/77] tests/t1701-rescue-fs wait for the device to appear.

After mkpart it needs to wait for the new partition to appear.

(cherry picked from commit 8d149b24eb2789fdf00743ef735e0cffde598040)
---
 tests/t1701-rescue-fs.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/t1701-rescue-fs.sh b/tests/t1701-rescue-fs.sh
index 9b726cf..bbf5eac 100644
--- a/tests/t1701-rescue-fs.sh
+++ b/tests/t1701-rescue-fs.sh
@@ -30,6 +30,7 @@ scsi_dev=$(cat dev-name)
     || { warn_ "$ME: no ext4 support"; Exit $fail; }
 
 parted -s $scsi_dev mklabel msdos mkpart primary ext2 1m 100%
+wait_for_dev_to_appear_ ${scsi_dev}1 || fail=1
 mkfs.ext4 ${scsi_dev}1 || { warn_ $ME: mkfs.ext4 failed; fail=1; Exit $fail; }
 
 # remove the partition
-- 
2.9.3

